frontend/server/app.ts

```diff
@@ -193,7 +193,11 @@ function createUIServer(options: UIConfigs) {
       }),
     );
   } else {
-    registerHandler(app.get, '/k8s/pod/logs', getPodLogsHandler(options.argo, options.artifacts));
+    registerHandler(
+      app.get,
+      '/k8s/pod/logs',
+      getPodLogsHandler(options.argo, options.artifacts, options.pod),
```

Comment:
Why is the `container name` being set via `options`? And the options name is `pod` instead of `container`. Won't you want to set container name as a request parameter?

Reply:
`options.pod` here is an object option for grouping the pod-related options, like `options.argo` and `options.artifacts`. All three of them are groups of options. currently the definition of options.pod is:
```
pod: {
    logContainerName: string;
}
```
Most of the time the container name for retrieving logs should be fixed. So in my change, I added an extra config in `UIConfigs` to specify the container name. And I grouped it under the `pod` sub-group.

Reply:
oh, so you want me to just pass the `pod.logContainerName`, as the third argument but not the pod sub-group?

Reply:
Sounds good on both comments, and yes you confirmed correct about the latter comment. I am wondering if there is any other flag you want to introduce under `options.pod`?  If not, maybe just passing `options.pod.logContainerName` to `getPodLogsHandler` is sufficient?

Reply:
Sure! It only passes the `options.pod.logContainerName` now.